EpsilonLights/Tests/tests/LightsTest/LightsTest.c

```diff
@@ -0,0 +1,391 @@
+#include "unity.h"
+
+#include "Mockstm32f4xx_hal_gpio.h"
+#include "Mockstm32f4xx_hal_can.h"
+//#include "Mockstm32f4xx_hal_adc.h"
+#include "Mockcmsis_os.h"
+
+//#include "stm32f4xx_hal_can.h"
+#include "Lights.h"
+#include "LightsTest.h"
+#include <stdio.h>
+
+lightsInfo lightCharacteristics = {0};
+
+uint8_t lightsInputs;
+uint8_t auxBmsInputs[2];
+uint8_t batteryErrors[5];
+uint8_t driversInputs[4];
+SigLightsHandle sigLightsHandle;
+
+osMutexId* canHandleMutex;
+
+void runLightsTests()
+{
+    RUN_TEST(test_updateLights1_headLightsOffBrakesLightsOffHazardsOffBmsStrobeLightsOff);
+    RUN_TEST(test_updateLights1_headLightsLowBreakLightsOnHazardsOnBmsStrobeLightOn);
+    RUN_TEST(test_updateLights1_headLightsHighHeadLightsLowBreakLightsOnHazardsOnBmsStrobeLightOn);
+    RUN_TEST(test_blinkSignalLights_sigLightsLeftOFFSigLightsRightOFF);
+    RUN_TEST(test_blinkSignalLights_sigLightsLeftONSigLightsRightONPrevSigStateZero);
+    RUN_TEST(test_blinkSignalLights_prevSigState0SigLightsLeftONSigLightsRightON);
```

Comment:
change this to test_blinkSignalLights_prevSigState1SigLightsLeftONSigLightsRightON, you have it right in the actual function but we probably forgot to change this here in the run test